MD-Verlet-Integrator/md_helper.py

```diff
@@ -1,15 +1,16 @@
-# Helper "Library" to compute Molecular Dynamics (MD) Trajectories
-# Numpy and OS python modules are required
-#
-# MD and Velocity Verlet Algorithms were taken from: :
-# Computational Soft Matter:  From Synthetic Polymers to Proteins, Lecture Notes,
-# Norbert Attig, Kurt Binder, Helmut Grubmuller, Kurt Kremer (Eds.),
-# John von Neumann Institute for Computing, Julich,
-# NIC Series, Vol. 23, ISBN 3-00-012641-4, pp. 1-28, 2004
-#
-# Created by: Leonardo dos Anjos Cunha
-# Date: 05/16/17
-# License: GPL v3.0
+"""
+Helper functions used to compute Molecular Dynamics (MD) trajectories
+
+References:
+Algorithms & equations taken from [Attig:2004].
+"""
+
+__authors__ = "Leonardo dos Anjod Cunha"
+__credits__ = ["Leonardo dos Anjod Cunha"]
```

Comment:
Name typo?

Reply:
👍 Good catch.  

I believe the conclusion was that for reference implementations, the citations should be included in multiple places, to avoid cluttering the code:
- Full list of proper citations for all scripts in the directory goes at the end of the directory's README, and
- `[LNFA:yyyy:pp]` keys pointing at the relevant citations should go anywhere within each script where it would be helpful to point at something, e.g., 
    - in the header pointing towards relevant papers containing equations and algorithmic details, 
    - in function docstrings implementing a particular equation, and
    - code comments nearby a particular equation or algorithmic choice.

I don't think it's explicitly necessary to reproduce the citation list within the script header if the keys for relevant citations is provided.  If there's a desire from folks that this be included, I can add them and make the relevant change to the contributor guidelines.